### DISCUSSION TITLE
Send new ou update product

### DISCUSSION THREAD

**Discussion by rsantosdosanjos** [abuzuhri/Amazon-SP-API-CSharp#424] [General]
When submitting a new product or updating how can I insert the DescriptionData information as it was done in the old api

**Comment by abuzuhri** [comment]
Submit product use feed and this part same in the new api

**Reply by rsantosdosanjos** [reply]
but I can't map the DescriptionData object
![image](https://user-images.githubusercontent.com/19205139/204621075-a4491b6b-9f08-48de-9ad3-707da82bee3b.png)

How can I map product attributes?

**Reply by rsantosdosanjos** [reply]
in the lib it is commented

![image](https://user-images.githubusercontent.com/19205139/204631017-2c84fa76-845a-4fe7-a292-dc9af3f09587.png)

**Reply by abuzuhri** [reply]
Yes as mentioned before General xml for everything will need big work form us it's better every developer do one part and added here to do all ,if you need this part then do it and submit pull request and we will add to nuget

**Reply by rsantosdosanjos** [reply]
I see, if I manage to map the object I'll be sending it to you, thanks.
